pnpm-workspace.yaml

```diff
@@ -1,2 +1,3 @@
 packages:
-  - "typescript/*"
\ No newline at end of file
+  - "typescript/packages/*"
```

Comment:
why?

couldnt you just leave this as before?
also, i think you need another * to grab all subdirs


```
  - "typescript/**"
```